### ISSUE TITLE
Great

### ISSUE THREAD

**Issue by AbleLincoln** [JuliaW1/Colmar-Academy#2]
Good job including alt attributes for a lot of your elements. As you said, this is really great for accessibility for screen-readers. Every site should have good accessibility, but it is something a lot of developers neglect. So good job including this.

Also, I like the new logo 🐏  😄 

https://github.com/JuliaW1/Colmar-Academy/blob/master/index.html#L20

**Comment by JuliaW1** [comment]
Thank you for your input.
